.github/ISSUE_TEMPLATE/general_topic_for_discussion.yml

```
@@ -0,0 +1,13 @@
+---
+name: General topic 
+description: Describe about the general topic that you want to discuss in weekly IRC community meeting.
```

```suggestion
description: Describe about the general topic that you want to discuss in weekly IRC community meeting.
```
@gundalow , as a native speaker, what do you think of removing `about` from the sentence?